app/code/Magento/ImportExport/Controller/Adminhtml/Import/Index.php

```diff
@@ -5,11 +5,15 @@
  */
 namespace Magento\ImportExport\Controller\Adminhtml\Import;
 
-use Magento\Framework\App\Action\HttpGetActionInterface as HttpGetActionInterface;
-use Magento\ImportExport\Controller\Adminhtml\Import as ImportController;
+use Magento\Framework\App\Action\HttpGetActionInterface;
+use Magento\Framework\App\Action\HttpPostActionInterface;
 use Magento\Framework\Controller\ResultFactory;
+use Magento\ImportExport\Controller\Adminhtml\Import as ImportController;
 
-class Index extends ImportController implements HttpGetActionInterface
+/**
+ * Import Index Controller
+ */
+class Index extends ImportController implements HttpGetActionInterface, HttpPostActionInterface
```

Comment:
Why should this action handle post requests?

Reply:
Thanks for review @ihor-sviziev 

Actually I'm still progressing on the pull request facing few more issues regards the replace addnotice deprecated methods. I will fix that too and then let you know.

Reply:
Ok, let me know once you’ll finish